tests/baselines/reference/deprecatedCompilerOptions4.errors.txt

```diff
@@ -0,0 +1,47 @@
+/foo/tsconfig.json(3,19): error TS5096: Flag 'ES3' is deprecated and will stop functioning in TypeScript 6.0. Specify 'ignoreDeprecations: "5.5"' to silence this error.
+/foo/tsconfig.json(4,9): error TS5096: Flag 'noImplicitUseStrict' is deprecated and will stop functioning in TypeScript 6.0. Specify 'ignoreDeprecations: "5.5"' to silence this error.
+/foo/tsconfig.json(5,9): error TS5096: Flag 'keyofStringsOnly' is deprecated and will stop functioning in TypeScript 6.0. Specify 'ignoreDeprecations: "5.5"' to silence this error.
+/foo/tsconfig.json(6,9): error TS5096: Flag 'suppressExcessPropertyErrors' is deprecated and will stop functioning in TypeScript 6.0. Specify 'ignoreDeprecations: "5.5"' to silence this error.
+/foo/tsconfig.json(7,9): error TS5096: Flag 'suppressImplicitAnyIndexErrors' is deprecated and will stop functioning in TypeScript 6.0. Specify 'ignoreDeprecations: "5.5"' to silence this error.
+/foo/tsconfig.json(8,9): error TS5096: Flag 'noStrictGenericChecks' is deprecated and will stop functioning in TypeScript 6.0. Specify 'ignoreDeprecations: "5.5"' to silence this error.
+/foo/tsconfig.json(9,9): error TS5096: Flag 'charset' is deprecated and will stop functioning in TypeScript 6.0. Specify 'ignoreDeprecations: "5.5"' to silence this error.
+/foo/tsconfig.json(10,9): error TS5096: Flag 'out' is deprecated and will stop functioning in TypeScript 6.0. Specify 'ignoreDeprecations: "5.5"' to silence this error.
+/foo/tsconfig.json(11,31): error TS5098: Invalid value for '--ignoreDeprecations'.
+
+
+==== /foo/tsconfig.json (9 errors) ====
+    {
+        "compilerOptions": {
+            "target": "ES3",
+                      ~~~~~
+!!! error TS5096: Flag 'ES3' is deprecated and will stop functioning in TypeScript 6.0. Specify 'ignoreDeprecations: "5.5"' to silence this error.

```

Comment:
Something weird is going on here; the only allowed `ignoreDeprecations` value is `"5.0"` and these features will cease to function in 5.5, not 6.0